README.md

```diff
@@ -1082,6 +1083,48 @@ Widevine (license url based):
     );
     _widevineController.setupDataSource(_widevineDataSource);
 
+```
+
+ClearKey:
+
+A ClearKey Mp4 file can be generated with MP4Box as follow:
+
+- Create drm_file.xml with the following contents.
+```xml
+<GPACDRM type="CENC AES-CTR">
+  <DRMInfo type="pssh" version="1">
+    <BS ID128="1077efecc0b24d02ace33c1e52e2fb4b"/>
+    <BS bits="32" value="1"/>
+    <BS ID128="cd7eb9ff88f34caeb06185b00024e4c2"/>
+  </DRMInfo>
+  <CrypTrack IV_size="8" first_IV="0xbb5738fe08f11341" isEncrypted="1" saiSavedBox="senc" trackID="1">
+    <key KID="f3c5e0361e6654b28f8049c778b23946" value="a4631a153a443df9eed0593043db7519"/>
+  </CrypTrack>
+   <CrypTrack IV_size="8" first_IV="0xbb5738fe08f11341" isEncrypted="1" saiSavedBox="senc" trackID="2">
+    <key KID="f3c5e0361e6654b28f8049c778b23946" value="a4631a153a443df9eed0593043db7519"/>
+  </CrypTrack>
+
+</GPACDRM>
+
+
+```
+- Create the mp4 container using  [MP4Box](https://gpac.wp.imt.fr/)
+  - MP4Box -crypt drm_file.xml  testvideo.mp4  -out testvideo_encrypt_tmp.mp4
+  - MP4Box -frag 240000 testvideo_encrypt_tmp.mp4 -out testvideo_encrypt.mp4 (need to create multi segment mp4 file as ExoPlayer does not read the pssh block on a single segment mp4 file)
+```dart
+ BetterPlayerDataSource(
```

Comment:
Could you format this dart code (please add ',' at the end of ')').

Reply:
Done.